### PR TITLE
Add subnets section

### PR DESCRIPTION
**Pull request by pnowosie**

### CODE REVIEW

**Review by DerTiedemann — APPROVED** [review]
LGTM

**Comment by pnowosie** [comment]
@peak3d I'd appreciate your look at this PR. Thank you 🙏

**Review by c4t-ag — APPROVED** [review]
LGTM!, Thx!

File: docs/subnets/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+### Separation of Concerns
+
+In a heterogeneous network of blockchains, some validators will not want to validate certain blockchains because they simply have no interest in those blockchains. The Subnet model allows validators to only concern themselves with blockchains that they care about. This reduces the burden on validators.
+
+### Application-Specific Requirements
+
+Different blockchain-based applications may require validators to have certain properties. Suppose there is an application that requires large amounts of RAM or CPU power. A Subnet could require that validators meet certain [hardware requirements](../apps/nodes/run-camino-node.md#requirements) so that the application doesn’t suffer from low performance due to slow validators.
+
+Subnets can have their own token economy with their own native tokens and fee markets. They can launch their own blockchains with customized virtual machines. See [Customize a Subnet](https://docs.avax.network/subnets/customize-a-subnet) for more details.
```
Comment (c4t-ag) [inline docs/subnets/README.md:43]: Pls. either remove sentence or create / link to our docs page....

File: docs/subnets/subnet-cli.md [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+--node-ids="NodeID-P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5" \
+--subnet-id="24tZhrm8j8GCJRE9PomW8FaeqbgGS4UAQjJnqqn8pq5NwYSYV1"
+```
+
+And its console log:
+
+```text
+2022-05-20T16:10:22.980-0600	info	client/client.go:81	fetching X-Chain id
+2022-05-20T16:10:22.981-0600	info	client/client.go:87	fetched X-Chain id	{"id": "qzfF3A11KzpcHkkqznEyQgupQrCNS6WV6fTUTwZpEKqhj1QE7"}
+2022-05-20T16:10:22.981-0600	info	client/client.go:96	fetching AVAX asset id	{"uri": "http://127.0.0.1:12913"}
```
Comment (c4t-ag) [inline docs/subnets/subnet-cli.md:341]: Is this really AVAX on Camino node??

File: docs/subnets/subnet-cli.md [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+2022-05-20T16:10:22.981-0600	info	client/client.go:105	fetched AVAX asset id	{"id": "BUuypiq2wyuLMvyhzFXcPyxPMCgSp7eeDohhQRqTChoBjKziC"}
+2022-05-20T16:10:22.981-0600	info	client/client.go:107	fetching network information
+2022-05-20T16:10:22.982-0600	info	client/client.go:116	fetched network information	{"networkId": 1337, "networkName": "network-1337"}
+
+Ready to add subnet validator, should we continue?
+*-------------------------*----------------------------------------------------*
+| PRIMARY P-CHAIN ADDRESS | P-custom18jma8ppw3nhx5r4ap8clazz0dps7rv5u9xde7p    |
+*-------------------------*----------------------------------------------------*
+| TOTAL P-CHAIN BALANCE   | 29,999,999.9000000 $AVAX                           |
```
Comment (c4t-ag) [inline docs/subnets/subnet-cli.md:350]: Should be CAM (+2 more)

File: docs/subnets/subnet-cli.md [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+2022-05-20T16:10:28.857-0600	info	platformvm/checker.go:48	polling P-Chain tx	{"txId": "gmrRTRK6671pBVopxHoPRx77hcvEvc5UKER1fFNgaRiFPY5Qh", "expectedStatus": "Committed"}
+2022-05-20T16:10:28.859-0600	info	poll/poll.go:42	start polling	{"internal": "1s"}
+2022-05-20T16:10:29.861-0600	info	poll/poll.go:66	poll confirmed	{"took": "1.002120951s"}
+added P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5 to subnet 24tZhrm8j8GCJRE9PomW8FaeqbgGS4UAQjJnqqn8pq5NwYSYV1 validator set (took 1.002120951s)
+
+waiting for validator P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5 to start validating 24tZhrm8j8GCJRE9PomW8FaeqbgGS4UAQjJnqqn8pq5NwYSYV1...(could take a few minutes)
+*-------------------------*----------------------------------------------------*
+| PRIMARY P-CHAIN ADDRESS | P-custom18jma8ppw3nhx5r4ap8clazz0dps7rv5u9xde7p    |
+*-------------------------*----------------------------------------------------*
+| TOTAL P-CHAIN BALANCE   | 29,999,999.8990000 $AVAX                           |
```
Comment (c4t-ag) [inline docs/subnets/subnet-cli.md:380]: Should be CAM

**Review by c4t-ag — CHANGES_REQUESTED** [review]
1 Link and a few AVAX -> CAM missing...

File: docs/subnets/subnet-cli.md (hunk 2 above)
Comment (pnowosie) [reply docs/subnets/subnet-cli.md]: No there isn't - this output I missed to replace with the real output from `camino-subnet-cli`

**Pull request merged**